test/db/esil/x86_32

```diff
@@ -2370,7 +2370,9 @@ CMDS=<<EOF
 # Common parts
 #
 e asm.arch=x86
-e asm.assembler=x86.as
+?==x86 `e asm.arch`
```

Comment:
also, let's assume a space between the command name and the arguments.
```suggestion
?== x86 `e asm.arch`
```
otherwise it becomes harder for the parser to know where the command ends and the arguments begins.

Reply:
do you want this space to be mandatory?

Reply:
I think yes. There should be a clear separation between commands and arguments (the same as you can't do `echoCiao` but you have to do `echo ciao`).